config/metasrv.example.toml

```diff
@@ -130,6 +130,25 @@ broker_endpoints = ["127.0.0.1:9092"]
 ## Otherwise, use topics named `topic_name_prefix_[0..num_topics)`
 auto_create_topics = true
 
+## Enable automatically WAL pruning.
+## Set to `true` to enable automatically WAL pruning to delete unused remote WAL entries periodically.
+auto_prune_topic_records = false
+
+## Interval of automatically WAL pruning.
+auto_prune_interval = "300s"
```

Comment:
Can we use a single option to control this, such as `auto_prune_topic_records_interval`, where 0 means never?